src/libraries/System.Private.CoreLib/src/System/Collections/Generic/HashSet.cs

```diff
@@ -811,8 +811,8 @@ public bool SetEquals(IEnumerable<T> other)
                 }
 
                 // Already confirmed that the sets have the same number of distinct elements, so if
-                // one is a superset of the other then they must be equal.
-                return ContainsAllElements(otherAsSet);
```

Comment:
After these changes, there is now only one caller for `ContainsAllElements(IEnumerable<T>)` (`IsSupersetOf`). Should the helper method just be inlined there?

Reply:
Sure